silkworm/db/datastore/snapshots/segment/kv_segment_test.cpp

```diff
@@ -36,8 +36,12 @@ struct CharCodec : public Codec {
         word.push_back(*byte_ptr_cast(&value));
         return word;
     }
-    void decode_word(Bytes& input_word) override {
-        value = *byte_ptr_cast(input_word.data());
+    void decode_word(Word& input_word) override {
+        auto input_word_view = input_word.byte_view();
```

Comment:
```suggestion
        ByteView input_word_view = input_word.byte_view();
```
I prefer to only use `auto` for long and complex types. With an explicit type the code is easier to understand (given that the type name itself is readable).